src/content/posts/releases/mitmproxy-11/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Mitmproxy 11: Full HTTP/3 Support"
+date: 2024-09-12
+weight: 10
+tags:
+  - gsoc
+  - releases
+authors:
+  - gaurav-jain
+---
+
+We are excited to announce the release of mitmproxy 11, which introduces full support for HTTP/3 in both transparent
+and reverse proxy modes. We're also bringing in a ton of DNS improvements that we'll cover in this blog post.
+
+<!--more-->
+
+##### *Editorial Note:*
+
+*Hi! I'm [Gaurav Jain], one of the students selected for this year's Google Summer of Code program to work on mitmproxy.
+During this summer, I've worked on improving various low-level networking parts of mitmproxy some of which include
+HTTP/3 and DNS. You can find my project report [here].*
+
+[Gaurav Jain]:  {{< relref "/authors/gaurav-jain" >}}
+[here]: https://gist.github.com/errorxyz/af6f26549e9122f3ff3b93fd9d257df1
+
+## HTTP/3
+
+HTTP/3 now "just works" for reverse proxies. Your mitmproxy instance will listen for
+both TCP and UDP packets and handle all HTTP versions thrown at it:
+
+```shell
+$ mitmproxy --mode reverse:https://http3.is
+```
+
+Our transparent proxy modes now all support HTTP/3 as well:
+
+```shell
+$ mitmproxy --mode wireguard
+$ mitmproxy --mode local
+$ mitmproxy --mode transparent
+```
+
+We have successfully tested HTTP/3 support with Firefox, Chrome, various cURL builds, and other clients to iron out 
+compatibility issues.
```

Comment:
just curious, what other clients have we tested on?

Reply:
aioquic, reqwest :)